example/tests/elementActions.test.js

```diff
@@ -330,5 +338,32 @@ describe("Element Actions", () => {
         expect(await resultElement.text()).toEqual(remotePath);
     });
 
+    //Skipping the Webkit for download tests until the issue is resolved https://github.com/microsoft/playwright/issues/5396
+    it.jestPlaywrightSkip({ browsers: ["webkit"] }, "should download a file when an absolute path is provided", async () => {
+        //Arrange
+        const filePath = localPath + "/example/examplePages/files/example.zip";
+        const resultFilePath = localPath + "/example/testFiles/temp/example.zip";
+        const downloadElement = new Element("#downloadLink");
+        await page.goto(`file:///${localPath}/example/examplePages/download.html`);
+
+        //Act
+        await downloadElement.downloadFile(resultFilePath, true);
+
+        //Assert
+        expect(await fs.readFile(filePath)).toEqual(await fs.readFile(resultFilePath));
+    });
+
+    it.jestPlaywrightSkip({ browsers: ["webkit"] }, "should download a file when an relative path is provided", async () => {
+        //Arrange
+        const filePath = localPath + "/example/examplePages/files/example.zip";
+        const resultFilePath = "/example/testFiles/temp/example.zip";
+        const downloadElement = new Element("#downloadLink");
+        await page.goto(`file:///${localPath}/example/examplePages/download.html`);
 
+        //Act
+        await downloadElement.downloadFile(resultFilePath, false);
+
+        //Assert
+        expect(await fs.readFile(filePath)).toEqual(await fs.readFile(process.cwd() + resultFilePath));
```

Comment:
perhaps localPath needs to be used instead of process.cwd() here?

Reply:
Yes. Fixed.